src/xtopcom/xdb/xdb.h

```diff
@@ -20,9 +20,9 @@ class xdb_error : public std::runtime_error {
 
 class xdb : public xdb_face_t {
  public:
-    explicit xdb(const std::string& name);
+    explicit xdb(const std::string& name, bool readOnly = false);
```

Comment:
for .h file, default value 'false' is not necessary?
it is specified on the call side?

Reply:
with default value，call side can necessary not provide the param， so compatible current code(without to add param)

Reply:
call side will pass the flag in?
![image](https://user-images.githubusercontent.com/2065762/123414378-65ad9080-d5e6-11eb-8ebd-85d83a112c55.png)